packages/ui/dialogs/my-account/index.tsx

```diff
@@ -398,14 +383,6 @@ class MyAccount extends Component<Props, State> {
         { !!emailTextInputErrorText && <Text style={style.warningText}>{emailTextInputErrorText}</Text>}
         <Button round onPress={submitEmail} text={updateAccountText} />
       </View>),
-      (<View style={style.spaceHorizontalL}>
-        <TouchableHighlight {...underlayColor} onPress={() => this.getPhoto()}>
-          <View style={general.centeredColumn}>
-            <Text style={[style.text, style.spaceTopL, style.center]}>{userPic ? accountManagement.changeProfilePic : accountManagement.addProfilePic}</Text>
```

Comment:
The English text for `addProfilePic` is  "Use Picture from Phone". It should be "Set Profile Pic".

However, how about using an icon button here instead of any text at all. Then we don't need localization and it will look better. We have an icon called "camera" or "mt-camera" that should already work with our `Button` class. It would be cool if it were placed over the profile pic (might need absolute positioning).

Reply:
With the icon present, we can also get rid of the `changeProfilePic` text, which in English is "Tap to Change"